app/components/maps/SidebarReplays.tsx

```diff
@@ -33,7 +33,7 @@ export const SidebarReplays = ({
     onRefreshReplays,
     selectedReplayDataIds,
 }: Props): JSX.Element => {
-    const defaultPageSize = 25;
+    const defaultPageSize = 14;
```

Comment:
14 is a bit of a weird number imo - 15 would be nicer, but that would probably require some layout changes?

Reply:
Yeah, 15 just doesn't fit, in order for it to fit, the padding of the buttons would need to be reduced even more. And since the user doesn't see the number 14 anyway I thought it would be fine (the dropdown for page size is removed).